### PR TITLE
Added a short paragraph about the background

### PR DESCRIPTION
**Pull request by Nyholm**
This will fix https://github.com/php-http/fig-standards/issues/17

@sagikazarmark, feel free to make changes to this PR.

### CODE REVIEW

File: proposed/http-client/http-client-meta.md [hunk 1]
```diff
@@ -66,6 +66,14 @@ One could be more granular when defining exception. For example, `TimeOutExcepti
 could be subtypes of `NetworkException`. The chosen approach is not to define such subtypes because
 the exception handling in a consuming library would not be different between those exceptions. 
 
+### Background
+
+The HTTP client PSR has been inspired and created by the [php-http team](https://github.com/orgs/php-http/people). 
+Back in 2015 they created HTTPlug which was an interface for HTTP clients. They wanted an abstraction 
+for third party libraries to rely on so nobody had to rely on Guzzle/Buzz directly. A stable version was tagged
```
Comment (dbu) [inline proposed/http-client/http-client-meta.md]: maybe "They wanted an abstraction that third party libraries can use to not rely on a specific HTTP client implementation like Guzzle 5, Guzzle 6 or Buzz."

**Comment by dbu** [comment]
what is the build error? do we need to rebase this branch on master?

File: proposed/http-client/http-client-meta.md (hunk 1 above)
Comment (Nyholm) [reply proposed/http-client/http-client-meta.md]: Thank you.

**Comment by Nyholm** [comment]
I've updated the text. Yes, that is a build error due invalid config. I guess @michaelcullum sees this and fixes it when he got the time.

**Pull request merged**